deutsches-archaologisches-institut.csl

```diff
@@ -23,17 +28,17 @@
         <group delimiter=". ">
           <choose>
             <if variable="author">
-              <names variable="editor">
+              <names variable="editor" font-variant="normal">
                 <name delimiter=" &#8211; " initialize-with=". "/>
-                <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+                <label form="short" strip-periods="true" prefix=" (" suffix=")"/>
               </names>
             </if>
           </choose>
           <choose>
             <if variable="author editor" match="any">
-              <names variable="translator">
+              <names variable="translator" font-variant="normal">
```

Comment:
You should again be able to delete the ` font-variant="normal"` here (which is the default).